views/home.njk

```diff
@@ -233,7 +233,7 @@
                 Want to talk about what Federalist can do for your agency? Questions about our ATO? Want to try it out for yourself?
               </p>
               <p>
-                <a class="usa-button" href="https://docs.google.com/forms/d/1iB8aW7c9r1QH3s8XElQCrnXRGjAiPUYpWG1CMeEqGIo/viewform">Get in touch</a>
+                <a class="usa-button" href="/contact/">Get in touch</a>
```

Comment:
Don't need this change given where this is on page - right @annahsebok?

Reply:
@line47 mind removing this change for now? I can put it back in later if we need.

Reply:
No problem

Reply:
as soon as this is changed back, I think this can be merged.